### ISSUE TITLE
Cleanup OpenJDK9 binaries

### ISSUE THREAD

**Issue by gdams** [AdoptOpenJDK/openjdk-build#167]
We need to cleanup the OpenJDK9 binaries that we produce as they are currently 180mb in size. We already do this [here](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/sbin/build.sh#L256-L260) for OpenJDK8

**Comment by neomatrix369** [comment]
@gdams where do we need to do this cleanup? not sure where this task applies, i'm missing the context here.

**Comment by karianna** [comment]
@neomatrix369 I think he's referring to cleaning them up on Jenkins.

**Comment by gdams** [comment]
no sorry, if you look at https://github.com/AdoptOpenJDK/openjdk-build/blob/master/sbin/build.sh#L256-L260 you'll notice that we remove some large files from the JDK before bundling it up as a tarball. We should look at doing the same for OpenJDK9 if possible as the tarballs are pretty big

**Comment by neomatrix369** [comment]
Go it @gdams, will do it asap on my list

**Comment by neomatrix369** [comment]
Looking at the issue...

**Comment by neomatrix369** [comment]
I found out that OpenJDK8 binaries were also affected, not just 9, @gdams does this sound like your observation as well?

**Comment by neomatrix369** [comment]
@gdams 
In OpenJDK8, the demo folder still contains:
```
  | jfc
  | jpda
  | jvmti
  | management
  | nbproject
  | nio/zipfs
  | scripting/jconsole-plugin
  | README
```
And makes up 6.7MB, I think we can let it sit there, or do you want it removed?

In OpenJDK9, the demo folder contains:
```
  | jfc
  | nbproject
  | README
```
The sizes of both the artifacts are still 80MB and 198MB respectively, lets discuss if there are any other components to remove.

**Comment by neomatrix369** [comment]
Closing issue as it is fixed via PR #172
